homeassistant/components/worldtidesinfo/sensor.py

```diff
@@ -33,6 +39,7 @@
         vol.Optional(CONF_LATITUDE): cv.latitude,
         vol.Optional(CONF_LONGITUDE): cv.longitude,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+        vol.Optional(CONF_DATUM): cv.string,
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler